pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -258,7 +258,7 @@ func getTier2StorageOptions(pravegaSpec *api.PravegaSpec) map[string]string {
 			"TIER2_STORAGE":        "EXTENDEDS3",
 			"EXTENDEDS3_BUCKET":    pravegaSpec.Tier2.Ecs.Bucket,
 			"EXTENDEDS3_URI":       pravegaSpec.Tier2.Ecs.Uri,
-			"EXTENDEDS3_ROOT":      pravegaSpec.Tier2.Ecs.Root,
```

Comment:
Not sure if you have to make any changes to pravega/docker/pravega/scripts/init_tier2.sh before this change . it uses ${EXTENDEDS3_ROOT})

Reply:
I just copy this comment into https://github.com/pravega/pravega/pull/4551,